libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -135,8 +135,10 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     float reserved2;
     float reserved3;
 
+    math::mat4f iblRotation; // contains the IBL's rotation
```

Comment:
Could we make it a mat3f?

Reply:
4d17478

Reply:
Why do you have to use `mat4f` in `PerViewUib`? `PerRenderableUib` stores `mat3f` matrices too.

Reply:
or is it because `PerRenderableUib` doesn't have to comform to std140 layout?

Reply:
Reverted in 5a7c586 after talking with Gabor. This is a cleaner version. In order to comply with std140, a mat3 should be laid out like this: each column of float3s should take up a float4's space, with float4 alignment. By having a `mat3` in `PerViewUib`, we have a hard time guaranteeing that.